.github/scripts/manage-images.py

```diff
@@ -0,0 +1,209 @@
+import os
+import json
+import grpc
+import math
+import logging
+import argparse
+from github import Github
+from datetime import datetime, timezone
+from yandex.cloud.compute.v1.image_pb2 import Image
+from yandex.cloud.compute.v1.image_service_pb2 import (
+    DeleteImageRequest,
+    ListImagesRequest,
+)
+from yandex.cloud.compute.v1.image_service_pb2_grpc import ImageServiceStub
+from yandexcloud import SDK, RetryInterceptor, backoff_linear_with_jitter
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s: %(levelname)s: %(message)s"
+)
+logger = logging.getLogger(__name__)
+
+# they won't appear in the list of images
+# but they will be protected from deletion anyway
+PROTECTED_IMAGE_IDS = [
+    "arl2kdkt7j8aorb1ib1h",
+    "arlf5b2mm2k1s4b71u4t",
+    "arlhviu5hj9aj6uqq6l3",
+    "arlpgqq65s5p13srj592",
+    "arl7p6s8d25e8m83kvi3",
+]
+
+
+def status_to_string(image: Image) -> str:
+    return Image.Status.Name(image.status)
+
+
+def convert_size(size_bytes):
+    if size_bytes == 0:
+        return "0B"
+    size_name = ("B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB")
+    i = int(math.floor(math.log(size_bytes, 1024)))
```

Comment:
From the doc: 
Help on built-in function floor in module math:

floor(x, /)
    Return the floor of x as an Integral.
    
    This is the largest integer <= x.
(END)
You probably forgot to remove int()